[MIG] stock_landed_costs_currency: Migration to 16.0

BT-jdziurzynski opened the pull request.

Commit:
[ADD] Add new module stock_landed_costs_currency

ps-tubtim committed:
[MIG] stock_landed_costs_currency: Migration to 13.0

BT-jdziurzynski committed:
[IMP] stock_landed_costs_currency: pre-commit stuff

BT-anieto commented on stock_landed_costs_currency/models/stock_landed_cost.py:
```diff
@@ -0,0 +1,24 @@
+        comodel_name="res.currency",
+        required=True,
+        related=None,
+        default=lambda self: self.env.user.company_id.currency_id,
```
Get the company from the environment like self.env.company.currency_id

BT-anieto reviewed (commented):
Hi @BT-jdziurzynski! Thanks for your PR. Just a small comment :)

BT-aleonard reviewed (approved):
Hi, thanks for migrating. 
The only thing I would add is to add yourself as a contributor.

rousseldenis commented:
/ocabot migration stock_landed_costs_currency

BT-jdziurzynski commented:
Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?

rousseldenis commented:
> Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?

@BT-jdziurzynski You can always inspect the lines that are missing in tests if you go to the 'Files changed' tab here.

![image](https://github.com/OCA/stock-logistics-workflow/assets/19529533/1a2c55a9-0d51-4d8e-adf3-f887593e8ec4)


So, adding a test that cover that line should greenify coverage.

For contributor problem, I don't get what you mean.

BT-jdziurzynski commented:
> > Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?
> 
> @BT-jdziurzynski You can always inspect the lines that are missing in tests if you go to the 'Files changed' tab here.
> 
> ![image](https://user-images.githubusercontent.com/19529533/251679338-1a2c55a9-0d51-4d8e-adf3-f887593e8ec4.png)
> 
> So, adding a test that cover that line should greenify coverage.
> 
> For contributor problem, I don't get what you mean.

I already extended the tests to cover these lines and codecov was satisfied. This changed after I added myself as a contributor.

rousseldenis commented:
> > > Hey @rousseldenis, I don't understand why codecov check is reduced and not passing after I added myself as a contributor. Could you take a look at this?
> > 
> > 
> > @BT-jdziurzynski You can always inspect the lines that are missing in tests if you go to the 'Files changed' tab here.
> > ![image](https://user-images.githubusercontent.com/19529533/251679338-1a2c55a9-0d51-4d8e-adf3-f887593e8ec4.png)
> > So, adding a test that cover that line should greenify coverage.
> > For contributor problem, I don't get what you mean.
> 
> I already extended the tests to cover these lines and codecov was satisfied. This changed after I added myself as a contributor.

@BT-jdziurzynski Apparently not as this is the result for last build

BT-jdziurzynski committed:
[16.0][MIG] stock_landed_costs_currency: Migration to 16.0

BT-anieto commented:
Hello @rousseldenis Tests are passing right now. Could it be that the needs_review tag is preventing the PR to be merged?
Shall anything else be done here? Thank you very much!

rousseldenis commented:
/ocabot merge nobump
